explorer/explorerroutes.go

```diff
@@ -1101,6 +1103,41 @@ func (exp *explorerUI) TxPage(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	// Check if tx is an atomic swap redemption tx or swap contract tx.
+	var swapType string
+	var atomicSwaps []*apitypes.AtomicSwap
+
+	// First check if this tx redeems 1 or more atomic swap contracts.
+	atomicSwaps, err = rpcutils.AtomicSwapsRedeemedInTx(exp.nodeClient, hash, exp.ChainParams)
```

Comment:
We could potentially skip this if 1) this is a stake tree transaction, or 2a) there are no p2sh outputs and 2b) there are no inputs with the correct sigscript type. If not now, then add a `TODO` here.